kartothek/io/testing/utils.py

```diff
@@ -20,13 +20,13 @@ def create_dataset(dataset_uuid, store_factory, metadata_version):
     df_list = [
         {
             "label": "cluster_1",
-            "data": [("core", df.copy(deep=True)), ("helper", df_helper)],
-            "indices": {"core": {val: ["cluster_2"] for val in df.TARGET.unique()}},
+            "data": [("table", df.copy(deep=True)), ("helper", df_helper)],
+            "indices": {"table": {val: ["cluster_2"] for val in df.TARGET.unique()}},
         },
         {
             "label": "cluster_2",
-            "data": [("core", df.copy(deep=True)), ("helper", df_helper)],
-            "indices": {"core": {val: ["cluster_2"] for val in df.TARGET.unique()}},
+            "data": [("table", df.copy(deep=True)), ("helper", df_helper)],
+            "indices": {"table": {val: ["cluster_2"] for val in df.TARGET.unique()}},
```

Comment:
Tiny nitpick: I would also use `SINGLE_TABLE` here so if it gets renamed tests do not need to be changed again (but some people may argue you shouldn't have that logic for a test)

Reply:
Changed it to `SINGLE_TABLE`